website/docs/guides/athena-qs.md

```diff
@@ -0,0 +1,341 @@
+
+:::tip Videos for you
+You can check out [dbt Fundamentals](https://learn.getdbt.com/courses/dbt-fundamentals) for free if you're interested in course learning with videos.
+:::
+
+### Prerequisites​
+
+- You have a [dbt Cloud account](https://www.getdbt.com/signup/). 
+- You have an [AWS account](https://aws.amazon.com/).
+- You have set up [Amazon Athena](https://docs.aws.amazon.com/athena/latest/ug/getting-started.html).
+
+### Related content
+
+- Learn more with [dbt Learn courses](https://learn.getdbt.com)
+- [CI jobs](/docs/deploy/continuous-integration)
+- [Deploy jobs](/docs/deploy/deploy-jobs)
+- [Job notifications](/docs/deploy/job-notifications)
+- [Source freshness](/docs/deploy/source-freshness)
+
+## Getting started
+
+Download these CSV files (the Jaffle Shop sample data) that you will need for this guide:
+    - [jaffle_shop_customers.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_customers.csv)
+    - [jaffle_shop_orders.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_orders.csv)
+    - [stripe_payments.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/stripe_payments.csv)
+
+## Set up an Amazon S3 bucket
+
+For the following guide you can use an existing S3 bucket or create a new one.
+
+In either case, download the following CSV files (the Jaffle Shop sample data) and load them into your S3 bucket:
+- [jaffle_shop_customers.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_customers.csv)
+- [jaffle_shop_orders.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_orders.csv)
+- [stripe_payments.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/stripe_payments.csv)
+
+
+## Configure Amazon Athena
+
+1. Log into your AWS account and navigate to the **Athena console**.
+    - If this is your first time in the Athena console (in your current AWS Region), click **Explore the query editor** to open the query editor. Otherwise, Athena opens automatically in the query editor.
+1. Open **Settings** and find the **Location of query result box** field.
+    1. Enter the path of the S3 bucket (prefix it with `s3://`).
+    2. Navigate to **Browse S3**, select the S3 bucket you created, and click **Choose**.
+1. **Save** the settings configurations
+1. In the **query editor**, create a database by running `create database YOUR_DATABASE_NAME`.
+1. To make the database you created the one you `write` into, select it from the **Database** list on the left side menu. 
+1. There are 2 options to access the Jaffle Shop data in the S3 bucket:
+    1. Manually create the tables
+    2. Create a glue crawler to recreate the data as external tables (recommended)
+1. Once the tables have been created, you will able to `SELECT` from them. 
+
+## Set up security access to Athena
+
+To setup the security access for Athena, determine which access method you want to use: 
+    1. The `aws_access_key_id` and `aws_secret_access_key` (recommended)
```

Comment:
Change to bullets because they're not steps they're a choice.
```suggestion
* Obtain `aws_access_key_id` and `aws_secret_access_key` (recommended)
```